account/identity.go

```diff
@@ -222,21 +228,55 @@ func (m *GormIdentityRepository) Query(funcs ...func(*gorm.DB) *gorm.DB) ([]*Ide
 	return objs, nil
 }
 
+// First returns the first Identity element that matches the given criteria
+func (m *GormIdentityRepository) First(funcs ...func(*gorm.DB) *gorm.DB) (*Identity, error) {
+	defer goa.MeasureSince([]string{"goa", "db", "identity", "first"}, time.Now())
+	var objs []*Identity
+	log.Debug(nil, map[string]interface{}{
+		"pkg":          "identity",
+		"identityList": objs,
```

Comment:
correct, that was a too fast copy/paste :/ I remove the `"identityList": objs,` key/value in the call to `Debug()`